litellm-proxy-extras/litellm_proxy_extras/schema.prisma

```diff
@@ -89,6 +89,7 @@ model LiteLLM_TeamTable {
 		team_id    String @id @default(uuid())
     team_alias  String? 
     organization_id String?
+    permission_id String?
     admins String[]
     members String[]
     members_with_roles Json @default("{}")
```

Comment:
Can you also add this to Organizations and User Tables

Reply:
Did you also mean `LiteLLM_EndUserTable`?